cli/keyring_list.go

```diff
@@ -0,0 +1,44 @@
+package cli
```

Comment:
todo: Please add a test file for this command like we did for the other keyring commands. This won't be tested in the integration tests and it's simple to do so within the `cli` package.

Reply:
Absolutely. This one was always on my to-do list. I now have it added.